tests/transports-integrations/run_all_tests.py

```
@@ -0,0 +1,339 @@
+        print(f"{'='*60}")
+        print(f"Description: {config['description']}")
+        print(f"Test file: {config['file']}")
+
+        start_time = time.time()
+
+        try:
+            # Run the test
+            cmd = [sys.executable, str(test_file)]
+
+            if verbose:
+                result = subprocess.run(
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)